libs/vkd3d/command.c

```diff
@@ -767,6 +767,11 @@ static HRESULT d3d12_fence_signal(struct d3d12_fence *fence, uint64_t physical_v
     return S_OK;
 }
 
+static void d3d12_fence_set_name(struct d3d12_fence *fence, const char *name)
+{
+    WARN("d3d12_fence_set_name not implemented\n");
```

Comment:
FIXME

Reply:
I think for this one we *could* technically name the timeline semaphores? Would need some extra code for when we create a new underlying semaphore.

Reply:
In theory, I guess, but it's not worth the effort I think.

Reply:
Actually, I don't think we create new underlying semaphores anymore, not after the refactor to virtual/physical timelines.

Reply:
I have just made this not anything for now. If we want this later we can implement it.